AppCenterReactNativeShared/ios/post-clone.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+echo "Installing pods for AppCenterReactNativeShared in `pwd`"
+
```

Comment:
Please add `pod repo update` before `pod install`.

Azure Pipeline mac VM doesn't always come with the latest pod repos.